src/read.c

```diff
@@ -3389,6 +3389,41 @@ static avifResult avifDecoderFlush(avifDecoder * decoder)
     return AVIF_RESULT_OK;
 }
 
+// Searches for the primary color item or the auxiliary alpha item.
```

Comment:
Document  `parentItemID`. Mention that `parentItemID` is ignored if `alpha` is false.